### PR TITLE
[Operator] Sign

### PR DESCRIPTION
**Pull request by ashutoshparkhi**
Support for Sign in relay.

Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

File: topi/tests/python/test_topi_math.py [hunk 1]
```diff
@@ -18,10 +18,11 @@ def test_ewise():
 
     shape = (20, 3)
 
-    def test_apply(func, name, f_numpy, low, high, check_round=False):
+    def test_apply(func, name, f_numpy, low, high, check_round=False, skip_name_check=False):
         B = func(A)
         assert tuple(B.shape) == tuple(A.shape)
-        assert B.op.body[0].name == name
+        if not skip_name_check:
```
Comment (srkreddy1238) [inline topi/tests/python/test_topi_math.py]: @ashutoshparkhi what is the name we are getting here ?

Comment (ashutoshparkhi) [reply topi/tests/python/test_topi_math.py]: sign is implemented with tvm::select inside TOPI. It errors out accessing the name:

AttributeError: '<class 'tvm.expr.Select'>' object has no attribute 'name'

Comment (srkreddy1238) [reply topi/tests/python/test_topi_math.py]: ok

**Comment by tqchen** [comment]
Thanks @srkreddy1238 @ashutoshparkhi this is now merged

**Pull request merged**